install/host_preparation.adoc

```diff
@@ -283,6 +283,11 @@ install the following package:
 # yum install openshift-ansible
 ----
 +
+[NOTE]
+====
+If you face an error while running `yum install openshift-ansible` command, see https://access.redhat.com/solutions/3949501.
```

Comment:
```suggestion
If you encounter an error while running `yum install openshift-ansible` command, see https://access.redhat.com/solutions/3949501.
```

Using the word "face" may have translation issues, I would advise using encounter.

Reply:
Addressed